test/v3/connectors/ListLossClaimsConnectorSpec.scala

```diff
@@ -165,7 +165,7 @@ class ListLossClaimsConnectorSpec extends LossClaimConnectorSpec {
         MockHttpClient
           .get(
             url = s"$baseUrl/income-tax/claims-for-relief/$nino",
-            parameters = Seq(("taxYear", "2019")),
+            parameters = Seq(("taxYearClaimedFor", "2019")),
             config = dummyIfsHeaderCarrierConfig,
             requiredHeaders = requiredIfsHeaders,
             excludedHeaders = Seq("AnotherHeader" -> "HeaderValue")
```

Comment:
In `listLossClaimsResult` helper method please rename `incomeSourceType` parameter to `typeOfLoss` (line 180)